docs/api/implementation-guide.md

```diff
@@ -0,0 +1,100 @@
+|[4.9](../release-notes/NuGet-4.9-RTM.md)|[Embedded licenses](#embedded-files)|
+|[5.3](../release-notes/NuGet-5.3.md)|[Embedded icons](#embedded-files) <br/> [Package deprecation in `RegistrationBaseUrl` (package metadata resource)](#known-vulnerability-and-deprecation-data)|
+|[5.8](../release-notes/NuGet-5.8.md)|[Package vulnerability information in `RegistrationsBaseUrl` (package metadata resource)](#known-vulnerability-and-deprecation-data)|
+|[5.10](../release-notes/NuGet-5.10.md)|[Embedded readme](#embedded-files)|
+|6.7|[PreAuthenticate authenticated requests](#url-structure-for-authenticated-feeds) <br/> [`VulnerabilityInfo` resource](#known-vulnerabilities-database-vulnerabilityinfo)|
+
+## Owner field
+
+Consider two of the [package manifest file (`.nuspec`)](../reference/nuspec.md) fields, `<authors>` and `<owners>`.
+Package authors who are packaging third-party content often put the third-party name in the `<authors>` field.
+The `<owners>` field was intended to denote who published the package on a repository, and therefore who should be contacted in case of packing issues or questions.
+
+This was explained in [a blog post from 2013](https://devblogs.microsoft.com/nuget/managing-package-owners/), so the `<owners>` field is considered deprecated the `.nuspec` file.
+If package's manifest contain this metadata, it should be ignored.
+Do not return the value of the `.nuspec` file's `<owners>` field in the `owners` property in the [search resource](./search-query-service-resource.md) or [package metadata resource](./registration-base-url-resource.md) JSON response.
+
+If your repository has per-package permissions, it is recommended to report the accounts that have permissions to publish new versions in the `owner` metadata for search and package metadata resources JSON responses.
+
+## Embedded files
+
+Package [icons](../reference/nuspec.md#icon), [license](../reference/nuspec.md#license), and [readme](../reference/nuspec.md#readme) files can be (and are recommended to be) embedded in the package.
+These files need a URL endpoint, either extracted and put on a static file server, or a URL that dynamically extracts the files from the `.nupkg` on request, so that they can be viewed without downloading the entire `nupkg`.
+If your package repository provides package browsing and viewing package details, you can use the URLs to show customers the embedded content on your website.
+
+Finally, the [package metadata resource](./registration-base-url-resource.md) and [search resource](./search-query-service-resource.md) must contain the hosted URL in the `iconUrl`, `licenseUrl`, and/or `readmeUrl` properties of the JSON response.
+Packages (`.nupkg` files) are immutable and must not be modified.
```

Comment:
This is not strictly true per protocol. Several feeds allow mutability today and it's more that client caches things in a couple tricky ways making package mutability problematic or at least harder to navigate for the package consumer. 

I'm okay with keeping it simple in this guide since suggesting they CAN be mutable would make it even worse, but such a blanket statement might give people the wrong impression. Maybe something like:

> NuGet.org and NuGet client software are designed around a concept of package immutability where a .nupkg for a given package version will never change. The embedded files mentioned in this section support that concept. There are however third-party package feeds which allow varying degrees of package immutability. As such, there is now hard requirement of package immutability but due to client-side caching and other factors, it is recommended that authors and feed maintainers treat packages as immutable. 

Again, feel free to scrap it but this statement might confuse users or feeds implementors which have evidence to the contrary.

Reply:
I made a different change that hopefully explains it well enough.  I'm particularly concerned about too much information and being technically pedantic over practical, which will make it harder for developers of multi-ecosystem package repositories (nuget + npm + pip, etc) that aren't interested in NuGet's nuances.